docs/source/logging/index.md

```diff
@@ -42,6 +42,19 @@ After setting the environment variable, any subsequent Kedro commands use the lo
 If the `KEDRO_LOGGING_CONFIG` environment variable is not set, Kedro will use the [default logging configuration](https://github.com/kedro-org/kedro/blob/main/kedro/framework/project/default_logging.yml).
 ```
 
+### Change the verbosity of specific parts of Kedro
+
+One can also customise logging at the runtime and redefine the logging configuration provided in the `logging.yml`.
+In the example below we change a logging level to `WARNING` for the `kedro.io.data_catalog` component logger specifically, the logging for the rest of the components will remain unchanged.
+The same can be done for higher/lower-level components without affecting the top-level.
+
+```python
```

Comment:
That's a good catch, I've added a clarification that this is relevant for the notebook environment. We cannot make it in `settings.py` since logging configuration from `logging.yml` is applied after `settings.py` configuration https://github.com/kedro-org/kedro/blob/413dbca46a9cb8e29fc1761564c4e13e2efe4d8e/kedro/framework/project/__init__.py#L273, so it will be redefined by what is provided in `logging.yml`

Reply:
So, the alternative is to customise `logging.yml` described in the neighbouring section.

Reply:
Ah I see, makes sense!